apps/web/pages/[user].tsx

```diff
@@ -180,7 +182,9 @@ export default function User(props: inferSSRProps<typeof getServerSideProps> & E
                   <Link
                     prefetch={false}
                     href={{
-                      pathname: `/${user.username}/${type.slug}`,
+                      pathname: newBookerEnabled
```

Comment:
Temporary. This way if you have the cookie, you directly go to the new page without having to change the url manually.